### DISCUSSION TITLE
Ownership of MiscPS3 on GitHub

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#957] [General]
Hey @aldostools, I'm the creator of the NixSDK/MirrorSDK repos which provide a prebuilt toolchains for different operating systems. For a couple of personal reasons, I've decided to distance myself from PS3 development and delete my current GitHub account. However, I would like these two repos two keep existing after I do so, since ManaGunZ uses NixSDK for automatic builds and MirrorSDK preserves, among other things, the movian toolchain which is not publicly available since movian.tv went down.

So, I wanted to ask if you would be willing to be an owner of the [MiscPS3](https://github.com/orgs/MiscPS3/repositories) GitHub organization, which I created to house these projects. The repos themselves wouldn't be directly tied to your account, and I've made sure to disable issues/discussion in each of them. Moreover, the NixSDK shouldn't break in the future since it doesn't download tarballs like the main toolchain repo, but rather has copies of every tarball archived, so it should be able to stand on its own for eternity (or, at least until Debian 11 docker images stop existing).

Also, sorry for posting this twice. I wasn't sure if you saw the message I left on the Resource page pull request.

**Comment by aldostools** [comment]
Mr @rhynec thank you for considering me for the ownership of your repository.

Honestly I'm not sure if I can handle more projects than the ones that I'm currently involved privately.

Maybe zarh, bucanero or Joonie86 could take care of your repository. They are devs with a great reputation in various scenes.
I suggest you to talk with them privately via their repo, twitter or psx-place forum.

**Reply by ghost** [reply]
Hey @aldostools! Sorry for the late reply, I've been busy with other stuff. Thank you for suggesting other developers that could take up the project! I'll look into it.
